packages/database/src/modelClasses/index.js

```diff
@@ -31,6 +31,7 @@ import { MapOverlayGroupModel } from './MapOverlayGroup';
 import { MapOverlayGroupRelationModel } from './MapOverlayGroupRelation';
 import { MapOverlayModel } from './MapOverlay';
 import { MeditrakDeviceModel } from './MeditrakDevice';
+import { MeditrakSyncQueueModel } from './MeditrakSyncQueue';
```

Comment:
I quite liked the idea that some models were entirely owned by a specific package, so didn't need to exist in the database package at all. Totally fine with this change though, and I see that it would be most devs' expectation that this would be an exhaustive list.

Reply:
I know what you mean, and I see the definitely see the value in that. Currently with all the typescript packages we limit which models they're aware of by having a custom ModelRegistry definition for each package, which seems to handle information hiding really well. But if there's a model with a whole lot of business logic, it does indeed feel a bit awkward if that logic is in say `@tupaia/database` if it really just relates to another package...

That being said the gradual pull of moving all the model classes (or atleast having a base model class) for each database table in `@tupaia/database` does seem to be happening... Ultimately I feel we may end up relying on something like the much discussed [@tupaia/types](https://linear.app/bes/issue/RN-509/add-basic-tupaiatypes-package) to manage this...